src/test/java/ome/system/utests/PrefsTest.java

```diff
@@ -100,13 +95,4 @@ public void testMissingFilesOk() {
         ctx.getProperty("test");
     }
 
-    @Test
-    public void testOmeroVersion() {
-        ctx = new PreferenceContext();
-        ctx.setLocation(new ClassPathResource("omero.properties"));
-        String v = ctx.getProperty("omero.version");
```

Comment:
This raises an interesting question of how version detection is going to work in the future. One option I can think of is that PreferenceContext add an property source which is basically `someClass.getPackage().getImplementationVersion()`. By default it uses itself. But when a class is set, then it uses _that_ class. Then the `PreferenceContext` constructor in blitz's spring files, could pass, e.g., `Entry.class` for the version number.

Reply:
"version handling" is something we need to go back.
I will add the comment to https://trello.com/c/B3dJaHhe/77-version

Reply:
Sounds good. But that might speak for just disabling this test rather than deleting it, since the above would mean that it would still be `PreferenceContext`s responsibility to provide the `omero.version` endpoint (just without using `omero.properties`)

Reply:
agree, I will force push a commit to disable it